### ISSUE TITLE
CS dršťka is not a swear word

### ISSUE THREAD

**Issue by AdamBajger** [shutterstock/List-of-Dirty-Naughty-Obscene-and-Otherwise-Bad-Words#184]
This is actually a common word describing beef stomach as an ingredient in meals. See "dršťková polévka". It is sometimes being used in a similar form, but when it is used as a swear word, the form of "držka" is used most fo the time, because the people swearing actually do not waste time writing it properly. Including the full form in this list will falsely filter out a lot of texts containing information about cooking, eating or traditional czech meals.

**Comment by mcepl** [comment]
I agree. “Dřšťka” (or “držka”) even in the meaning “ugly face” as a swearword, which is the naughtiest meaning of the word, is an expressive (“Rozbil jsem mu dršťku/držku”), but not a bad word in the meaning of this list.
